hnn_core/cells_default.py

```diff
@@ -230,9 +308,16 @@ def _cell_L5Pyr(override_params, pos=(0.0, 0.0, 0), gid=0.0):
     )
 
 
-def _get_basket_soma(cell_name):
+def _get_basket_soma(cell_name, v_init=-64.9737):
```

Comment:
```suggestion
def _get_basket_soma(cell_name, v_init=-64.9737):
```
Nitpick but is there any reason to have 4 decimal points here rather than 2 like most of the other voltages?